hnn_core/gui/gui.py

```diff
@@ -854,6 +863,11 @@ def _get_poisson_widget(name, tstop_widget, layout, style, location, data=None,
                        layout=layout,
                        style=style)
 
+    synch_inputs = Checkbox(value=sync_evinput,
```

Comment:
```suggestion
    sync_inputs = Checkbox(value=sync_evinput,
```

just to be a little more consistent :)